ubuntu22.04/precompiled/Dockerfile

```diff
@@ -48,28 +48,21 @@ RUN if [ -n "${CVE_UPDATES}" ]; then \
         rm -rf /var/lib/apt/lists/*; \
     fi
 
-# update pkg cache and install pkgs for userspace driver libs
-RUN apt-get update && apt-get install -y --download-only --no-install-recommends nvidia-driver-${DRIVER_BRANCH}-server \
```

Comment:
Where else was `apt-get remote or apt-get clean` run after this step which was causing the packages to be removed? `apt-get update` itself doesn't remove downloaded packages.

Reply:
So when I downloaded packages using `apt-get install --download-only` and then ran an update `apt-get update`. I noticed that the downloaded packages in `/var/cache/apt/archives` were wiped out